opencraft/urls.py

```diff
@@ -26,18 +26,15 @@
 from django.contrib import admin
 from django.views.generic.base import RedirectView
 
-import instance
-
 
 # URL Patterns ################################################################
 
 urlpatterns = [
     url(r'^grappelli/', include('grappelli.urls')),
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^api/', include('api.urls', namespace="api")),
-    url(r'^instance/', include('instance.urls', namespace="instance")),
-    url(r'^beta/', include('betatest.urls', namespace='beta')),
+    url(r'^api/', include('api.urls', namespace='api')),
+    url(r'^instance/', include('instance.urls', namespace='instance')),
     url(r'^email-verification/', include('email_verification.urls', namespace='email-verification')),
     url(r'^favicon\.ico$', RedirectView.as_view(url='/static/img/favicon/favicon.ico', permanent=False)),
-    url(r'^$', instance.views.index),
+    url(r'^', include('registration.urls', namespace='registration')),
```

Comment:
Would it make sense to put the registration on its own url (like `/register`) and just redirect the home there, so we don't need to move it again when it gets a proper home page?